packages/cardhost/app/components/card-manipulator.js

```diff
@@ -145,4 +179,46 @@ export default class CardManipulator extends Component {
   delete() {
     this.deleteCard.perform();
   }
+
+  @action
+  beginDragging(field, mousedownEvent) {
+    let dragState;
+
+    set(this, 'finishDrag', (dropEvent) => {
```

Comment:
is there a reason we we add a new function to the component as opposed to just setting this function's contents in the `dropField()` action on line 222?

Reply:
this is a copypaste from some experimentation with `ember-animated`, which needed the `field` and the initial `mousedownEvent` in the `drop` event to do math-y stuff. Since we aren't (yet) using animations, this could certainly be simplified as you suggested.